### DISCUSSION TITLE
aria-hidden console.error

### DISCUSSION THREAD

**Discussion by herzaso** [floating-ui/floating-ui#2511] [General]
I use [jest-fail-on-console](https://www.npmjs.com/package/jest-fail-on-console) which fails my jest tests if console is being used, but in a specific scenario, I get the following `console.error`:
```
aria-hidden <ref *1> HTMLDivElement {
...
} in not contained inside HTMLBodyElement {
...
} . Doing nothing
```

I've noticed that there is a `aria-hidden` package that `floating-ui` uses that throws [this error](https://github.com/theKashey/aria-hidden/blob/9220c8f4a4fd35f63bee5510a9f41a37264382d4/src/index.ts#L34) but my application seems to work fine. What does it mean?

**Comment by atomiks** [comment]
Not sure without seeing any code, but if you upgrade your `@floating-ui/react` version that won't appear anymore regardless

**Reply by herzaso** [reply]
Thanks! Upgrading the package did it 🙌
